src/plugins/CoreLoadPlugin.ts

```diff
@@ -38,15 +44,26 @@ const jsMidPattern = /\.(t|j)sx?$/;
  */
 function isContextual(module: NormalModule, issuers: string[]): boolean {
 	const { rawRequest, userRequest } = module;
-	const relative = /^\.(\.*)\//;
 	const request = userRequest.replace(/\.[a-z0-9]+$/i, '');
-	return relative.test(rawRequest) && issuers.some((issuer: string) => path.resolve(issuer, rawRequest) === path.resolve(request));
+	return relativePathPattern.test(rawRequest) && issuers.some((issuer: string) => path.resolve(issuer, rawRequest) === path.resolve(request));
+}
+
+/**
+ * @private
+ * Strips the drive prefix from a file path and normalizes the path separator to '/'.
+ *
+ * @param path The path on which to operate.
+ *
+ * @return The normalized file path.
+ */
+function normalizeFilePath(path: string): string {
+	return path.replace(/\\/g, '/').replace(/^[cC]:/, '');
```

Comment:
couldn't the users machine be using any letter for the windows installation? not just `c` ?.
Aren't there filepath resolving util libraries available for this?

Reply:
Good catch; the drive can be any letter. I'd copied that from elsewhere in the module. To answer your question, `path` methods do not convert from `\` to `/`.